test/graphgym/my_config.yaml

```diff
@@ -0,0 +1,5 @@
+defaults:
+  - dataset: KarateClub
+  - transform@dataset.transform:
+      - NormalizeFeatures
```

Comment:
This is super cool. I think I'm missing a lot of context here. How does these names transfer to a function?

Reply:
The names are derived from `torch_geometric.transforms`, see [here](https://pytorch-geometric.readthedocs.io/en/latest/modules/transforms.html). That is `torch_geometric.transforms.NormalizeFeatures` will get mapped to the following `@dataclass`:
```python
@dataclass
class NormalizeFeatures:
    _target_: "torch_geometric.transforms.NormalizeFeatures"
    attrs: List[str] = ["x"]
```
This pattern is also used [torch-hydra](https://github.com/pytorch/hydra-torch), a project from the PyTorch team, see, *e.g.*, [here](https://github.com/pytorch/hydra-torch/blob/main/hydra-configs-torch/hydra_configs/torch/optim/adam.py).